index.ts

```diff
@@ -7,10 +7,13 @@ export async function main() {
   // inputs
   const stainless_api_key = getInput('stainless_api_key', { required: true });
   const input_path = getInput('input_path', { required: true });
-  const output_path = getInput('output_path', { required: true });
+  const output_path = getInput('output_path');
 
   const raw_spec = await loadSpec(input_path);
   const decoratedSpec = await decorateSpec(raw_spec, stainless_api_key);
+  if ((output_path == undefined || output_path == null) && output_path != '') {
```

Comment:
```suggestion
  if (!output_path) {
```

we wouldn't want to write to an empty filepath, right?